src/test/java/com/mentoring/RegistrationTest.java

```diff
@@ -30,4 +41,139 @@ public void successfulRegistration() {
         //TODO: Add notification assertion
         softAssertion.assertAll();
     }
+
+    private void assertInvalidUserName(MainPage mainPage) {
+        softAssertion.assertThat(mainPage.isUserNameInvalid())
+                .as(X_ICON_MESSAGE + "username.")
+                .isTrue();
+    }
+
+    private void assertInvalidPassword(MainPage mainPage) {
+        softAssertion.assertThat(mainPage.isPasswordInvalid())
+                .as(X_ICON_MESSAGE + "password.")
+                .isTrue();
+    }
+
+    private void assertInvalidEmail(MainPage mainPage) {
+        softAssertion.assertThat(mainPage.isEmailAddressInvalid())
+                .as(X_ICON_MESSAGE + "e-mail address.")
+                .isTrue();
+    }
+
+    private void assertSubmitButtonEnabled(MainPage mainPage) {
+        softAssertion.assertThat(!mainPage.isSubmitButtonEnabled())
```

Comment:
In addition to the renaming, I suggest to change this statement to be true and set the assertion (expected value) to false.
So rather use softAssertion.assertThat(mainPage.isSubmitButtonEnabled()) and .isFalse at the end.
In general it's a good practice not to negate statements. It's easier to read, and it's more straightforward. Still there are cases when negating the statement makes more sense (e.g: much shorter code), but this is not one of those :)